packages/block-library/src/paragraph/block.json

```diff
@@ -36,6 +36,9 @@
 		"fontSize": true,
 		"lineHeight": true,
 		"__experimentalSelector": "p",
-		"__unstablePasteTextInline": true
+		"__unstablePasteTextInline": true,
+		"typography": {
```

Comment:
The shape that is part of WordPress 5.5 is:
	"supports": {
		"anchor": true,
		"className": false,
		"lightBlockWrapper": true,
		"__experimentalColor": {
			"linkColor": true
		},
		"__experimentalFontSize": true,
		"__experimentalLineHeight": true,
		"__experimentalFeatures": {
			"typography": {
				"dropCap": true
			}
		},
		"__experimentalSelector": "p",
		"__unstablePasteTextInline": true
	}

typography.dropCap is under __experimentalFeatures. I think we should avoid exposing typography.dropCamp under supports without any experimental mark when we know that we want to remove this from block.json. Block authors may replicate our json file.